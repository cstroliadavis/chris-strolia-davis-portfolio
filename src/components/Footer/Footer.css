footer {
    align-items: center;
    background-color: #556f70;
    bottom: 0;
    flex-direction: column;
    height: var(--footer-height);
    justify-content: center;
    position: absolute;
    width: 100%;
}

footer > .container {
    display: grid;
    grid-template-columns: 1fr 1fr 1fr;
    height: 50px;
    margin: 10px auto auto;
    max-width: 200px
}

footer > .container a {
    background-color: white;
    display: block;
    max-height: 40px;
    max-width: 40px;
}

footer > .container a > img {
    height: 36px;
    margin: 2px auto auto;
    width: 36px;
}
