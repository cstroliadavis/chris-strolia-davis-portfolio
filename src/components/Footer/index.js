import './Footer.css'
import githubLogo from '../../assets/img/GitHub-Mark-120px-plus.png';
import linkedInLogo from '../../assets/img/In-Blue-128.png';
import mediumLogo from '../../assets/img/Medium-Symbol-Black-RGB@1x.png';

export default function Footer() {
  return (
    <footer>
      <div className="container">
        <a href="https://github.com/cstroliadavis" target="github">
          <img src={ githubLogo } alt="Github Profile"/>
        </a>
        <a href="https://www.linkedin.com/in/cstroliadavis/" target="linkedin">
          <img src={ linkedInLogo } alt="Linked in Profile"/>
        </a>
        <a href="https://medium.com/@cstroliadavis" target="medium">
          <img src={ mediumLogo } alt="Medium Profile"/>
        </a>
      </div>
    </footer>
  );
}
