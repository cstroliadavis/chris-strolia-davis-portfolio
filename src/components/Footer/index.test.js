import { render, cleanup } from '@testing-library/react';
import Footer from './index'

afterEach(cleanup);

describe('Footer', () => {
  it('renders', () => {
    render(<Footer/>)
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<Footer/>);

    expect(asFragment()).toMatchSnapshot();
  });
});
