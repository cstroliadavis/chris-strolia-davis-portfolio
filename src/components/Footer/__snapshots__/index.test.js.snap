// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<DocumentFragment>
  <footer>
    <div
      class="container"
    >
      <a
        href="https://github.com/cstroliadavis"
        target="github"
      >
        <img
          alt="Github Profile"
          src="GitHub-Mark-120px-plus.png"
        />
      </a>
      <a
        href="https://www.linkedin.com/in/cstroliadavis/"
        target="linkedin"
      >
        <img
          alt="Linked in Profile"
          src="In-Blue-128.png"
        />
      </a>
      <a
        href="https://medium.com/@cstroliadavis"
        target="medium"
      >
        <img
          alt="Medium Profile"
          src="Medium-Symbol-Black-RGB@1x.png"
        />
      </a>
    </div>
  </footer>
</DocumentFragment>
`;
